oneview_redfish_toolkit/api/storage.py

```diff
@@ -128,6 +128,9 @@ def build_for_composed_system(server_profile, server_hardware_type,
             attrs["Drives"].append({
                 "@odata.id": attrs["@odata.id"] + "/Drives/" + drive_id
             })
+        if len(sas_logical_jbods) != 0:
```

Comment:
I thought "Volumes" should be added only if you have logical drives in the server profile. Not sure if it makes sense present Volumes when there are only sas logical JBODs. @bsoutham, @sidneifranco  what do you think?